File: iOS/Objects/src/B4iProject/LS_frmsplash.m

#import "iCore.h"
@interface LS_frmsplash:NSObject
@end

@implementation LS_frmsplash

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[views get:@"pnlheader"].Height = (int)((33 / 100.0 * height));
[views get:@"pnlprogress"].Left = (int)((50 / 100.0 * width) - ([views get:@"pnlprogress"].Width / 2));
[views get:@"pnlprogress"].Top = (int)((83 / 100.0 * height) - ([views get:@"pnlprogress"].Height / 2));
[views get:@"pb1"].Left = (int)(([views get:@"pnlprogress"].Width)/2.0 - ([views get:@"pb1"].Width / 2));
[views get:@"pnllogo"].Left = (int)((50 / 100.0 * width) - ([views get:@"pnllogo"].Width / 2));
[views get:@"pnllogo"].Top = (int)((38 / 100.0 * height) - ([views get:@"pnllogo"].Height / 2));

}
@end

File: Android/Objects/src/com/apps/demo/designerscripts/LS_frmsplash.java
package com.apps.demo.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmsplash{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("pnlheader").vw.setHeight((int)((33d / 100 * height)));
views.get("pnlprogress").vw.setLeft((int)((50d / 100 * width) - (views.get("pnlprogress").vw.getWidth() / 2)));
views.get("pnlprogress").vw.setTop((int)((83d / 100 * height) - (views.get("pnlprogress").vw.getHeight() / 2)));
views.get("pb1").vw.setLeft((int)((views.get("pnlprogress").vw.getWidth())/2d - (views.get("pb1").vw.getWidth() / 2)));
views.get("pnllogo").vw.setLeft((int)((50d / 100 * width) - (views.get("pnllogo").vw.getWidth() / 2)));
views.get("pnllogo").vw.setTop((int)((38d / 100 * height) - (views.get("pnllogo").vw.getHeight() / 2)));

}
}